package.json

```diff
@@ -88,12 +88,12 @@
     "prepack": "npm run build"
   },
   "devDependencies": {
-    "@playwright/test": "1.33.0",
+    "@playwright/test": "1.34.3",
```

Comment:
I'm not sure why 3 tests consistently fail when run on Chromium 114 (they pass on v113). @keithamus Any ideas?

I'll pin it back for now.